src/main/io/osd.c

```diff
@@ -484,6 +485,28 @@ static uint16_t osdConvertRSSI(void)
     return constrain(getRSSI() * 100 / RSSI_MAX_VALUE, 0, 99);
 }
 
+/**
+* Converts temperature into a string based on the current unit system
+* postfixed with a symbol to indicate the unit used.
+* @param temperature Raw temperature (i.e. as taken from getCurrentTemperature() in degC)
+*/
+static void osdFormatTemperatureSymbol(char *buff, float temperature)
+{
+    int units_symbol;
+    switch ((osd_unit_e)osdConfig()->units) {
+        case OSD_UNIT_IMPERIAL:
+            units_symbol = SYM_TEMP_F;
+            temperature = (temperature * (9/5)) + 32;
```

Comment:
Your calculation will always yield `temperature + 32`, since `9/5` will be evaluated to `1` at compile time.

```suggestion
            temperature = (temperature * (9.0f/5)) + 32;
```

Reply:
Thanks for the pointer!